kernel/comps/virtio/src/transport/mmio/device.rs

```diff
@@ -201,7 +201,7 @@ impl VirtioTransport for VirtioMmioTransport {
         self.common_device.io_mem().slice(0x100..0x200)
     }
 
-    fn device_features(&self) -> u64 {
+    fn get_device_features(&self) -> u64 {
```

Comment:
Could you use the verb _read_ or _load_ instead of _get_? The former ones sound more heavy-weight than the latter one.